### ISSUE TITLE
BNF-Tabelle “Schlagwort_Titel_GK_überarbeitet” bitte in Muscat einspielen

### ISSUE THREAD

**Issue by Docudoctor** [rism-international/tasks#855]
BNF-Tabelle “Schlagwort_Titel_GK_überarbeitet” bitte in Muscat einspielen:

https://docs.google.com/spreadsheets/d/1XRS9zLqngOPpKACTQfmlc38TX7rYNTY4dIniUK1Pmho/edit#gid=910655293

Danke,
Guido

**Comment by jenniferward** [comment]
Leere Felder sollen ignoriert werden (d.h. nicht überschreiben). Nur die Felder mit Text importieren.

**Comment by HirschSt** [comment]
Sollen Inhalte gleich welcher Art überschrieben werden? Sollen Einträge mit Semikolon aufgesplittet werden?

**Comment by jenniferward** [comment]
_Sollen Einträge mit Semikolon aufgesplittet werden?_ 
Ja bei 650$a, 240$m, 594$b, 594$c.

**Comment by jenniferward** [comment]
@Docudoctor Könntest du bitte die erste Frage von Stephan beantworten:
Sollen Inhalte gleich welcher Art, falls in der Tabelle vorhanden, überschrieben werden?

**Comment by Docudoctor** [comment]
> 
> 
> Sollen Inhalte gleich welcher Art, falls in der Tabelle vorhanden, überschrieben werden? Sollen Einträge mit Semikolon aufgesplittet werden?

Ja. Inhalte in Muscat, wenn überhaupt vorhanden, sollen anhand der vorliegenden Tabelle überschrieben werden.
